app/views/layouts/_user_menu.html.haml

```diff
@@ -16,6 +16,11 @@
     = link_to(conference_program_tracks_path(@conference.short_title)) do
       %span.fa.fa-road
       My Tracks
+-if @conference && @conference.program.cfps.for_booths.try(:open?)
```

Comment:
It was a thoughtless mistake. In order to make it shown when there are no any requests, I forgot that it should also be shown after the cfb is over.